backend/cmd/server.go

```diff
@@ -69,6 +69,8 @@ func main() {
 	*pluginDir = defaultPluginDir(*pluginDir)
 	log.Printf("plugins-dir: %s\n", *pluginDir)
 
+	go watchForChanges(*pluginDir)
```

Comment:
Should we add this only if we're not in-cluster? Just thinking it's unlikely that in-cluster deployments will update the plugins without relaunching the service.
ATM either way seems fine and the extra IO should not be too much, but I'd prefer to err on the side of caution and have it the monitor only locally.

Another option is to add it only if the `-dev` option is passed, but we'd have to expose that to the desktop app's CLI, and later on we'd likely have to revisit that since we may want to allow desktop users to just replace the plugins on the fly (not only for development but for regular use, I mean).

Reply:
Indeed. I think -in-cluster option is best, and have changed it to not watch when in-cluster is on. I thought about -dev too, and had the same thought about having it on by default with the app.

So yeah... turning it off for in-cluster it is :)